docs/qs_first_boot.md

```diff
@@ -0,0 +1,101 @@
+
+**On Device Setup**
+
+If this option is selected the next screen will have a list of avaliable WiFi connections
+
+![](https://github.com/OpenVoiceOS/ovos_assets/blob/master/Images/Screenshot%20-%20Select%20wifi%20on-device.png?raw=true)
+
+Select your desired network, and enter a password if needed
+
+![](https://github.com/OpenVoiceOS/ovos_assets/blob/master/Images/Screenshot%20-%20Insert%20wifi%20password.png?raw=true)
+
+If everything went correctly, you should be connected to the internet and after a short period of time, OVOS is loading skills that require internet, you will be presented with the homescreen.
+
+![](https://github.com/OpenVoiceOS/ovos_assets/blob/master/Images/Screenshot%20-%20Horizontal%20homescreen.jpg?raw=true)
+
+**Configure WiFi after skipping intial setup**
+
+If you have skipped the Wi-Fi network setup before, or you just want to reconfigure your network. On the homescreen of your OpenVoiceOS device swipe down the top menu and click the "Wi-Fi" icon. This brings you to the same on-device configuration screen.
+
+![](https://raw.githubusercontent.com/OpenVoiceOS/ovos_assets/master/Images/Screenshot%20-%20Network%20connections.png)
+
+From here you can select another network or click the configuration icon on the right of connected network for details or to remove it from the configured networks.
+
+![](https://raw.githubusercontent.com/OpenVoiceOS/ovos_assets/master/Images/Screenshot%20-%20Connection%20details.png)
+
+**Mobile Setup**
+
+If this option is chosen, you will be prompted to connect your mobile device, or computer to the hotspot `OVOS`
+
+![](https://github.com/OpenVoiceOS/ovos_assets/blob/master/Images/Screenshot%20-%20Connect%20hotspot.png?raw=true)
+
+The rest of this option coinsides with the headless image WiFi setup.  [Continue Mobile WiFi Setup](#WiFi-1)
+
+## Raspbian-ovos - Headless
+If you are trying to run OVOS on a RPi3, this is the image to use.  It **DOES NOT** provide a GUI, therefore some things that are avaliable on the buildroot image, are not avaliable here.
+
+Once again, it may take several minutes for this first boot to complete.  OVOS resizes the partition on the drive, and loads the required plugins.
+
+#### Ethernet
+There should be no issues connecting automatically if your router accepts DHCP requests
+
+#### WiFi
+On first boot, you should hear a voice prompt instructing you to connect to the `OVOS` hotspot
+
+> "Open a browser to 'start dot openvoiceos dot com'"
+
+Connect your mobile device or computer to the `OVOS` hotspot and open the webpage http://start.openvoiceos.com **NOTE** This is NOT the official OVOS website, rather a "Hotspot" created by OVOS and will be removed after the device is connected to WiFi
```

Comment:
```suggestion
Connect your mobile device or computer to the `OVOS` hotspot and open the webpage [start.openvoiceos.org](https://start.openvoiceos.org/). **NOTE** This is *not* the official OVOS website, instead a "Hotspot" created by OVOS and will be removed after the device is connected to WiFi.
```

I also have a tendency to use `http` instead of `https` for sites that support `http` since it allows older clients to connect without issue, but for sites that auto-redirect to `https` anyway, it slows the connection process.

Additionally,

> This is *not* the official OVOS website, instead a "Hotspot"

doesn't explain what it is. What's a hotspot? A user should probably be informed of this.

However, https://start.openvoiceos.org/ redirects me to https://openvoiceos.org/ now, so is that paragraph even necessary?

Reply:
I believe it IS necessary because the hotspot that is created does not go to that site.  it is `http://start.openvoiceos.com`  Note the `.com` not `.org`

But I did put a note on what a "HotSpot" is